sentinel/src/transitions/registration.js

```diff
@@ -69,7 +69,7 @@ const booleanExpressionFails = (doc, expr) => {
     } catch (err) {
       // TODO should this count as a fail or as a real error
       logger.warn('Failed to eval boolean expression:');
-      console.log(err);
+      logger.error(err.toString());
```

Comment:
You've upgraded this from a `log` to an `error` - let's stay with `info` for now...